packages/connect/src/index.ts

```diff
@@ -30,15 +30,19 @@ class ConnectRaw extends Connection {
 }
 
 export const connect = {
-  s3free: ({ blockstore }: Connectable) => {
-    const upload = 'https://udvtu5wy39.execute-api.us-east-2.amazonaws.com/uploads'
-    const download = 'https://crdt-s3uploadbucket-dcjyurxwxmba.s3.us-east-2.amazonaws.com'
-    const connection = new ConnectS3(upload, download)
+  s3free: ({ blockstore }: Connectable, databasename: string) => {
```

Comment:
instead of changing existing ones for now, lets make an awsFree that is like s3Free plus ws

Reply:
this is the main one to fix